package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ember-cp-validations",
+  "name": "@qonto/ember-cp-validations",
```

Comment:
is that compatible with `@ember-intl/cp-validations`?

Reply:
The idea is to use this one in `@ember-intl/cp-validations` actually. See https://github.com/ember-intl/cp-validations/pull/75#issuecomment-1017136214 for more context.